gui_qt/app.py

```
@@ -4407,6 +4474,31 @@ def _on_finished(self, exit_code: int):
+            manifest_path = self._writeback_manifest_path
+            self._update_writeback_from_check(
+                "\n".join(self._recheck_output_lines),
+                exit_code,
+                manifest_path,
+            )
+            if exit_code == 0:
+                self._refresh_workflow_from_latest_manifest()
+            self._active_command = ""
+            self._set_task_running(False)
+            if exit_code != 0:
+                self.statusBar().showMessage("重新检查失败，请查看诊断日志。", 8000)
```

Ruff (0.15.20)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)